evalml/tests/pipeline_tests/test_pipeline_utils.py

```diff
@@ -591,7 +591,7 @@ class MockRegressionPipeline(RegressionPipeline):
         generate_pipeline_code([Imputer(), LogisticRegressionClassifier()])
 
 
-def test_generate_code_pipeline_json_errors():
+def test_generate_code_pipeline_json_with_objects():
```

Comment:
No longer errors :)

With that, just kept one or two tests of different types of objects :)